src/coreclr/tools/Common/Internal/Metadata/NativeFormat/Generator/ReaderGen.cs

```diff
@@ -20,6 +20,7 @@ public void EmitSource()
         WriteLine("#pragma warning disable 649");
         WriteLine("#pragma warning disable 169");
         WriteLine("#pragma warning disable 282 // There is no defined ordering between fields in multiple declarations of partial class or struct");
+        WriteLine("#pragma warning disable IDE0059");
```

Comment:
What unnecessary assignments are being generated?  They can't be easily fixed instead?

Reply:
Every last assignment like https://github.com/dotnet/runtime/blob/main/src/coreclr/tools/Common/Internal/Metadata/NativeFormat/NativeFormatReaderGen.cs#L10283. The file looks committed from someone's machine and I didn't want to mess with regenerating it as part of this change.